### PR TITLE
Adds skeleton

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We need the standard Vidsy 💀.

### Solution

* Add it.

### Notes

* None.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine
```
Comment (revett) [inline Dockerfile]: Possible to version this?

Annoying that we can't use the Vidsy one.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
-# gqlgen
-(Binary) Container with the https://github.com/99designs/gqlgen binary
+<h1 align="center">gqlgen</h1>
+
+<p align="center">
+  Container with the [99designs/gqlgen](https://github.com/99designs/gqlgen) binary installed and ready to use.
```
Comment (revett) [inline README.md]: Need a HTML link here.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+</p>
+
+# Usage
+
+The container is pushed up to Docker Hub as [vidsyhq/gqlgen](https://cloud.docker.com/u/vidsyhq/repository/docker/vidsyhq/gqlgen).
+
+# Example
+
+```bash
+docker run --rm=true -v $PWD:/go/src/github.com/[org]/[repo name] --workdir=/go/src/github.com/[org]/[repo name]
```
Comment (revett) [inline README.md]:
```suggestion
docker run --rm=true -v $PWD:/go/src/github.com/[org]/[repo] --workdir=/go/src/github.com/[org]/[repo]
```

File: Dockerfile (hunk 1 above)
Comment (stevenjack) [reply Dockerfile]: Well the Vidsy one is `go-builder` which has a number of other things in it

**Pull request merged**